arduino-ide-extension/src/browser/serial/monitor/monitor-utils.ts

```diff
@@ -67,3 +67,7 @@ export function truncateLines(
   }
   return [lines, charCount];
 }
+
+export function linesToMergedStr(lines: Line[]) : string {
+  return lines.map((line: Line) => {return line.message}).join("");
+}
```

Comment:
```suggestion
export function linesToMergedStr(lines: Line[]): string {
  return lines.map((line: Line) => line.message).join('');
}

```


A simple `stringifyLines` or `joinLines` would be a more appropriate function name, maybe

Reply:
See [da8803e](https://github.com/arduino/arduino-ide/pull/2718/commits/da8803e7b1f1ff6bbcdb17e25f1c88832bed0089) and [5033769](https://github.com/arduino/arduino-ide/pull/2718/commits/503376947c3dff3837f25f4d9142fcbc0c1523bc)